### ISSUE TITLE
Integrate Query Monitor

### ISSUE THREAD

**Issue by hellofromtonya** [KnowTheCode/UpDevTools#1]
Query Monitor needs to be integrated into the suite. Remember, we want UpDevTools to be a complete integrated development solution, meaning one plugin to install on all of your development websites. Query Monitor is one of those necessary tools.

**Comment by jhabdas** [comment]
@hellofromtonya hello from josh. I learned about this dev kit from a joe you're training and used them to build my first WordPress plugin. And I want to tell you, as a developer with 20 years of JavaScript, moving into this toolkit made PHP development feel almost effortless.

One thing I missed while creating my plugin were some [little gotchas](https://github.com/wp-id/hyperdrive/issues/19) snuck up during beta testing, and I'm not sure how I missed them but that's my constructive feedback to help you improve this tool. Thank you so much for putting this together. I look forward to trying it when I dive into Laravel.

**Comment by Mte90** [comment]
any updates for this?

**Comment by hellofromtonya** [comment]
@Mte90 Nope, I haven't spent time integrating Query Monitor into the suite of tools.  For now, use the Query Monitor plugin separately as it's supported and works well.
